activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -475,7 +475,7 @@ def num_waiting_in_queue # :nodoc:
         @available.num_waiting
       end
 
-      # Return connection pool's usage statistic
+      # Returns the connection pool's usage statistic
       # Example:
```

Comment:
This one is public. :+1:  We can tidy it up a bit though:

```suggestion
      # Returns the connection pool's usage statistic.
```